### DISCUSSION TITLE
Failed to connect frontend to the backend in theia

### DISCUSSION THREAD

**Discussion by YOSSTM** [eclipse-theia/theia#11085] [Q&A] [answered]
Hello :) 

I am using theia to connect backend service to the frontend , right now I am just trying to show a simple string message, I do not have any compilation problem but while I try to launch the electron , the electron start but I am not able to see any UI.The problem disappear if I comment all the modification I did in the frontend part to connect to the service , I followed the theia extension related the backend service deployed by theia 
<img width="983" alt="electron" src="https://user-images.githubusercontent.com/94178254/165717869-81584175-af43-4dfe-b86f-d59f2ba5b98d.PNG">

**Comment by msujew** [comment, marked as answer]
Hi @YOSSTM,

Do any errors appear in the Chrome dev tools which might indicate the source of the error? You can open them using `Ctrl+Shift+I`/`Ctrl+Alt+I`.

**Comment by YOSSTM** [comment]
hello @msujew 

I tried both option  but I cannot open the chrome dev tools

**Comment by YOSSTM** [comment]
succeed to open the chrome dev tools 
I got this error:
index.js:112 Error: No native build was found for platform=win32 arch=x64 runtime=electron abi=98 uv=1 libc=glibc node=16.5.0 electron=15.3.7 webpack=true
    loaded from: C:\Users\ghouaiey\dev\apps\studio\electron

**Comment by YOSSTM** [comment]
Find the origin of the problem 
thanks @msujew
